src/InventoryManager.cpp

```diff
@@ -39,17 +40,31 @@ int InventoryManager::userInput() {
             std::cin.clear();
             std::cin.ignore(10000, '\n');
 
-            std::cout << "Name Perishable/NonPerishable SubCategory Quantity ID SalePrice "
-                         "CostOfGood Tax ExpirationDate (Enter -1000000 if unsure): ";
-            std::cin >> name >> category >> sub_category >> quantity >> id >> sale_price >> buy_price >> tax >>
-                expiration;
+            std::cout << "Enter item name: ";
+            std::cin >> name;
+            std::cout << "Enter item category (Perishable or NonPerishable): ";
+            std::cin >> category;
+            std::cout << "Enter item sub-category: ";
+            std::cin >> sub_category;
+            std::cout << "Enter item quantity: ";
+            std::cin >> quantity;
+            std::cout << "Enter item id: (up to 6 digits) ";
```

Comment:
I'd remove the up to 6 digits part because I don't think there's any validation for this anywhere or any reason to limit our values for ID.